Feature/mypage card components

mintnaka opened the pull request:
## PR 유형
- [x] 새로운 기능 추가
- [x] CSS 등 사용자 UI 디자인 변경
- [x] 코드 리팩토링

## PR 체크리스트
- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
![image](https://github.com/user-attachments/assets/032c7967-2642-446d-b406-5c256d0b477c)

#34

mintnaka committed:
[Style] 마이페이지 오타 변경

mintnaka committed:
Merge branch 'develop' of https://github.com/FRONTENDBOOTCAMP-12th/HobbyDobby-3rd into feature/mypage-card-components

mintnaka committed:
[Feat] 마이페이지 카드 컴포넌트 1
- 불꽃 아이콘 파일 추가
- 카드 컴포넌트 폴더, 파일 생성
- 카드 컴포넌트 스타일 추가
- 마이페이지 프로필헤더 및 프로필카 컴포넌트화

#26

mintnaka committed:
[Feat] 마이페이지 카드 컴포넌트 1
- 불꽃 아이콘 파일 추가
- 카드 컴포넌트 폴더, 파일 생성
- 카드 컴포넌트 스타일 추가
- 마이페이지 프로필헤더 및 프로필카 컴포넌트화

mintnaka committed:
Merge branch 'feature/mypage-card-components' of https://github.com/FRONTENDBOOTCAMP-12th/HobbyDobby-3rd into feature/mypage-card-components

somin2352 reviewed (changes requested):
- my-page-card.css 에서 img 태그로 스타일링한 부분도 클래스명을 지정해주시거나 중첩해서 넣어주세요
- ProfileCardProps, ProfileHeader, StatCardProps는 id 타입이 정의되어 있는데 함수 props는 전달받지 않았어요

mintnaka committed:
[Modify] 코드리뷰 반영 수정
- 카드 컴포넌트 props에서 사용 안 하는 id 속성 제거했습니다.
- css img 태그 스타일 card 클래스 안으로 중첩 처리했습니다.

somin2352 reviewed (approved):
확인 완료했습니다~
